modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -120,6 +121,10 @@ def validate
 
         private
 
+        def enable_vmbs_access?
+          form_attributes['recordConsent'] && (form_attributes['consentLimits'].nil? || form_attributes['consentLimits'].empty?)
```

Comment:
this
```
(form_attributes['consentLimits'].nil? || form_attributes['consentLimits'].empty?)
```
can be replaced with this
```
form_attributes['consentLimits'].blank?
```

its a rails method (opposite of `.present?`) that checks for
- nil
- empty string
- empty array
really anything that is "not something"